x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.tsx

```diff
@@ -208,8 +206,8 @@ export const PolicyDetails = React.memo(() => {
                   defaultMessage: 'Back to endpoint hosts',
                 }
               ),
-              href: formatUrl(hostListRouterPath),
-              pageId: SecurityPageName.administration,
+              href: '',
```

Comment:
Why is this empty?

Reply:
Because the `href` here works a little bit different, is like the path inside the page, not the absolute href, it is built inside the component. So here it has to be empty, meaning `/administration/endpoints`
I will change the href to optional in the backOptions type, so it is less weird.